StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityFlowView.swift

```diff
@@ -28,15 +28,19 @@ class IdentityFlowView: UIView {
         )
         static let buttonSpacing: CGFloat = 10
         static let buttonInsets = NSDirectionalEdgeInsets(
-            top: 8,
+            top: 4,
             leading: 16,
             bottom: 8,
             trailing: 16
         )
+        static let buttontopInsets = NSDirectionalEdgeInsets(
```

Comment:
not sure if it applies here, but adding a note of `directionalLayoutMargins` on UIStackView in case it is helpful (I tend to use that a lot even for just padding as its easier than autolayout):

https://github.com/stripe/stripe-ios/blob/89af67e0/StripeFinancialConnections/StripeFinancialConnections/Source/Native/AccountPicker/AccountPickerSelectionRowView.swift#L110-L116

Reply:
hmm it seems it does't work on buttonTopBackgroundView, which is an UIView instead of UIStackView